### PR TITLE
Add secrethub/exec example to README

### PR DESCRIPTION
**Pull request by florisvdg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,35 @@ Use this orb to load secrets from SecretHub into your CircleCI jobs.
 
 ## Usage
 
-To execute a command with secrets loaded from SecretHub, install the CLI and set it as the `shell` of the `run` command:
+To execute a command that needs secrets, replace your CircleCI `run` command with `secrethub/exec`.
+You can make secrets available to your command as environment variables by referencing their SecretHub path, prefixed by `secrethub://`:
```
Comment (SimonBarendse) [inline README.md:21]: Maybe introduce first what this orb does, then explain how to do it? How about switching these two lines?

File: README.md [hunk 2]
```diff
@@ -17,7 +17,35 @@ Use this orb to load secrets from SecretHub into your CircleCI jobs.
...
+      - image: cimg/base:stable
+    environment:
+      AWS_ACCESS_KEY_ID: secrethub://company/app/aws/access_key_id
+      AWS_SECRET_ACCESS_KEY: secrethub://company/app/aws/secret_access_key
+    steps:
+      - checkout
+      - secrethub/exec:
+          command: |
+            echo "This value will be masked: $AWS_ACCESS_KEY_ID"
+            echo "This value will be masked: $AWS_SECRET_ACCESS_KEY"
```
Comment (SimonBarendse) [inline README.md:40]: Do we want to explain this in the example code or explain it in text somewhere?

It now makes the example a bit more complex.

File: README.md [hunk 3]
```diff
@@ -17,7 +17,35 @@ Use this orb to load secrets from SecretHub into your CircleCI jobs.
...
+            ./deploy-my-app.sh
+workflows:
+  deploy:
+    jobs:
+      - deploy
+```
+
+You can also set the `shell` of the native CircleCI `run` command:
```
Comment (SimonBarendse) [inline README.md]:
```suggestion
Alternatively, you can set the `shell` of the native CircleCI `run` command:
```

File: README.md [hunk 4]
```diff
@@ -46,7 +74,8 @@ workflows:
       - deploy
 ```
 
-To load secrets from SecretHub into other orbs, install the CLI and set it as the `shell` of the `job`:
+You can either set it the `run` command level, on you can set it on the `job` level.
```
Comment (SimonBarendse) [inline README.md]:
```suggestion
You can either set the shell on the `run` command level, or you can set it on the `job` level to use it for every step in the job.
```

File: README.md (hunk 2 above)
Comment (florisvdg) [reply README.md:40]: Yes, we have the `Masking` section for that ⬇️

File: README.md (hunk 1 above)
Comment (florisvdg) [reply README.md:21]: That's how it is now, right? `execute a command that needs secrets` on line 1 is the 'what', `by referencing their SecretHub path` on line 2 is the 'how'.

Comment (SimonBarendse) [reply README.md:21]: Yes, you're right.

I was looking for a bit longer explanation than "execute a command that needs secrets", but maybe that's enough.

File: README.md (hunk 2 above)
Comment (SimonBarendse) [reply README.md:40]: Oke, so shall we remove it from the examples to make them clearer?

**Review by SimonBarendse — APPROVED** [review]
This one is still open, but not a blocker: https://github.com/secrethub/secrethub-orb/pull/10#discussion_r387696041

File: README.md (hunk 2 above)
Comment (florisvdg) [reply README.md:40]: No, it also fulfills the function of possibly preventing a major wtf-moment when seeing an `echo` of an `AWS_SECRET_ACCESS_KEY`.

**Pull request merged**